clang/lib/CIR/CodeGen/CIRGenExprConstant.cpp

```diff
@@ -290,9 +378,23 @@ mlir::Attribute ConstantEmitter::tryEmitPrivate(const APValue &value,
     cgm.errorNYI("ConstExprEmitter::tryEmitPrivate member pointer");
     return {};
   }
-  case APValue::LValue:
-    cgm.errorNYI("ConstExprEmitter::tryEmitPrivate lvalue");
+  case APValue::LValue: {
+
+    if (value.getLValueBase()) {
```

Comment:
We're probably going to want to leave ConstantLValueEmitter::tryEmit() as a separate function.

Reply:
Yes, i think later we need to upstreaming ConstantLValueEmitter and also implement the visitors